### ISSUE TITLE
GuildMemberManager#fetch() will timeout if you send an array containing values with letters

### ISSUE THREAD

**Issue by Vaylle** [discordjs/discord.js#4995]
**Please describe the problem you are having in as much detail as possible:**

When you use guild.members.fetch({user: Array}) , it will timeout if any of the value in the array contain anything else than numbers

**Include a reproducible code sample here, if possible:**

```js
    // This code will work fine even if the 2 last values are invalid
    const userIds = ['751069534424924266','321321321','00000000'];
    message.guild.members.fetch({user: userIds})
    .then(console.log)
    .catch(console.log);

    // This code will timeout :
    const userIds = ['ezaeza'];
    message.guild.members.fetch({user: userIds})
    .then(console.log)
    .catch(console.log);
```

Error after the timeout : 
```
Error [GUILD_MEMBERS_TIMEOUT]: Members didn't arrive in time.
    at /home/hivemine/node_modules/discord.js/src/managers/GuildMemberManager.js:314:16
    at Timeout._onTimeout (/home/hivemine/node_modules/discord.js/src/client/BaseClient.js:83:7)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7) {
  [Symbol(code)]: 'GUILD_MEMBERS_TIMEOUT'
}
```
**Further details:**

- discord.js version: discord.js@12.3.1
- Node.js version: v12.18.4
- Operating system: Ubuntu 20.04 LTS
- Priority this issue should have – please be realistic and elaborate if possible: Low

**Relevant client options:**

- partials: N/A
- gateway intents: N/A
- other: N/A

- [ ] I have also tested the issue on latest master, commit hash:

**Comment by bala41288** [comment]
I had the same problem. I reached this problem after several days of exploring. The root problem that I had was the users were not cached after my bot restarts. Then I thought fetching all the users on bot start would be a good solution to update the cache. While doing so, today I ran into same error message. After a few hours of investigation, I found this article. 

https://discordjs.guide/popular-topics/intents.html

Ideally they should have mentioned about this in a better way somewhere in this [page](https://discordjs.guide/additional-info/changes-in-v12.html).

![image](https://user-images.githubusercontent.com/23168216/98538378-ad2e1900-22b0-11eb-990f-35544596a42b.png)

I guess after the recent update they have intentionally switched off the feature to fetch the members of a guild. I then went to developers portal and switched that feature on.

![image](https://user-images.githubusercontent.com/23168216/98538515-dfd81180-22b0-11eb-971f-8e7a406a484b.png)

Even after switching it on, the caching is not automatic. But at least the below code is not giving me a timed out error message. 

`client.guild.members.fetch()`

In my case, I'm fetching all the users from my guild and not passing any user parameters. 

Hope this helps.
